apps/etl-func/src/func.ts

```diff
@@ -107,6 +121,30 @@ const main = async (config: Config) => {
     //we need to start the ingestion from this date
     startFromTime: "2023/01/01T00:00:00Z",
   });
+
+  // NOTE: we don't want to start the ingestion yet
```

Comment:
if we use the test container I think we can uncomment this right? In this way we can also do a test run if we want